pkg/helmcertifier/checks/checks.go

```diff
@@ -62,7 +67,23 @@ func HasReadme(uri string) (Result, error) {
 }
 
 func ContainsTest(uri string) (Result, error) {
-	return notImplemented()
+	c, err := loadChartFromURI(uri)
+	if err != nil {
+		return Result{}, err
+	}
+
+	containTest := false
```

Comment:
What if `Result` is initialized here?

```go
r := Result{Reason: ChartTestFilesDoesNotExists}
```

Later on in line 79:

```go
r.Ok = true
r.Reason = ChartTestFileExists
```

And in line 85:

```go
return r, nil
```

Reply:
done